daemons/gptp/common/ptp_message.cpp

```diff
@@ -959,9 +962,8 @@ void PTPMessageFollowUp::processMessage(IEEE1588Port * port)
 	master_local_freq_offset += 1.0;
```

Comment:
I must admit, I had to double check negative exponent rules. See https://en.wikipedia.org/wiki/Exponentiation#Negative_exponents. To my way of reading it
X * 2^(-41) == X / 2^ 41
so line 961
```
master_local_freq_offset /= 2ULL << 41;
```
should be correct. Am I missing something?